learn/core_concepts/relevancy.md

```diff
@@ -124,11 +131,11 @@ To add a custom ranking rule, you have to communicate the attribute name followe
 
 **The attribute must have either a numeric or a string value** in all of the documents contained in that index.
 
-Add this rule to the existing list of ranking rules using the [update ranking rules endpoint](/reference/api/ranking_rules.md#update-ranking-rules).
+You can add this rule to the existing list of ranking rules using the [update settings](/reference/api/settings.md#update-settings) or [update ranking rules endpoint](/reference/api/ranking_rules.md#update-ranking-rules).
```

Comment:
Same as a previous comment. I think what throws me off is that we have two options (update settings and update ranking rules), but only qualify one of them as an endpoint ("update settings" vs "update ranking rules endpoint.")

```suggestion
You can add this rule to the existing list of ranking rules using the [update settings](/reference/api/settings.md#update-settings) or [update ranking rules endpoint](/reference/api/ranking_rules.md#update-ranking-rules).
```

Reply:
It should be endpoint here too. I must have copy-pasted the text and link everywhere 🫠